server/models/skills.model.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AllowNull,
+  AutoIncrement,
+  Column,
+  CreatedAt,
+  Length,
+  Model,
+  PrimaryKey,
+  Table,
+  UpdatedAt,
+} from 'sequelize-typescript';
+
+interface SkillsAttributes {
+  skill_id: number;
+  skill_name: string;
+  created_at: Date;
+  updated_at: Date;
+}
+
+@Table({
+  tableName: 'Skills',
+  timestamps: true,
+})
+
+class Skills extends Model<SkillsAttributes> implements SkillsAttributes {
```

Comment:
models should be singular because they describe one thing 
tables should be plural because they house multiple of the same type of things

so for the `Skills` model, change it to `Skill`
same goes for the file name